src/pages/product-details/product-submit/product-submit.js

```diff
@@ -8,22 +8,27 @@ import FavouriteBorderIcon from '@material-ui/icons/FavoriteBorder';
 import Button from '@material-ui/core/Button';
 import { useStyles } from './product-submit.styles';
 
+import Toast from '../../../components/toast';
+
 import {
   addItemToWishlist,
   removeItemFromWishlist
 } from '../../../redux/wishlist/wishlist.actions';
 import { addItemToCart } from '../../../redux/cart/cart.actions';
+import { setToastMessage } from '../../../redux/toast/toast.actions';
 
 import {
   PDP_BUTTONS,
   TOOLTIPS
 } from '../../../translations/product-details.translations';
 
+import { TOAST_MESSAGE } from '../../../translations/toast.translations';
+
 const ProductSubmit = ({ setSizeIsNotSelectedError, sizes }) => {
   const styles = useStyles();
   const dispatch = useDispatch();
   const { language, productToSend, product, wishlistItems } = useSelector(
-    ({ Language, Products, User, Wishlist }) => ({
+    ({ Language, Products, User, Wishlist, Toast }) => ({
       language: Language.language,
```

Comment:
Please move this selector to separate file.

Reply:
done

Reply:
done